app/App.tsx

```diff
@@ -12,14 +12,15 @@ function App(): ReactElement {
   const [displayPage, setdisplayPage] = useState(PageDisplay.Authentication);
   return (
     <AppStyling>
-      {displayPage === PageDisplay.Authentication && (
+      <TabNavigation/>
+      {/* {displayPage === PageDisplay.Authentication && (
         <Authentication
           onLoginSuccess={() => {
             setdisplayPage(PageDisplay.TabNavigation);
           }}
         />
       )}
-      {displayPage === PageDisplay.TabNavigation && <TabNavigation />}
+      {displayPage === PageDisplay.TabNavigation && <TabNavigation />} */}
```

Comment:
was this meant to be committed or just for testing only

Reply:
it's for testing only

Reply:
yep generally should not commit stuff that's for testing - eg I'll remove all console logs from my code before I make a PR

Reply:
also an smaller-change way to do this would be to set the initial state of `displayPage` to something else

Reply:
ok i see the fix for this got mixed up in the other commit